.circleci/config.yml

```diff
@@ -19,13 +19,19 @@ aliases:
         ignore: /.*/
       branches:
         only: main
+  android-machine-emulator: &android-machine-emulator
+    executor:
+      name: android/android-machine
+      resource-class: large
+      tag: 2024.04.1
 
 orbs:
-  android: circleci/android@0.2.1
+  android: circleci/android@2.5.0
   gcp-cli: circleci/gcp-cli@2.1.0
   swissknife: roopakv/swissknife@0.65.0
   revenuecat: revenuecat/sdks-common-config@2.1.0
   macos: circleci/macos@2.3.2
+  flutter: circleci/flutter@2.1.0
```

Comment:
I think we probably should move more of our jobs to use this orb... But for now, I wanted to minimize the changes.